gnucash/gnucash-core-app.hpp

```diff
@@ -70,6 +70,8 @@ class CoreApp
     char *sys_locale = nullptr;
 };
 
-void gnc_load_scm_config(void);
+typedef void (*MessageCb) (const char *msg);
```

Comment:
That's the C way. Use [`std::function`](https://en.cppreference.com/w/cpp/utility/functional/function) instead and use lambdas instead of named callback functions.